scripts/sync-and-test-bucket.sh

```diff
@@ -32,6 +32,10 @@ fi
 # For previews, name the destination bucket with the PR number, to reduce the number of
 # buckets we create and to facilitate shorter sync times.
 destination_bucket="$(origin_bucket_prefix)-$(build_identifier)"
+if [[ "$GITHUB_WORKFLOW" == "Build and deploy new account" ]]; then
```

Comment:
Same `if, else-if, else` pattern. Or make `destination bucket` something the CI builds and passes in directly.